dm/dm/dm-ansible/scripts/DM-Monitor-Professional.json

```diff
@@ -3716,12 +3716,12 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "dm_relay_binlog_file{instance=~\"$instance\", node=\"relay\"} - ON(instance, task) dm_syncer_binlog_file{instance=~\"$instance\", task=~\"$task\", node=\"syncer\"}",
+          "expr": "dm_relay_binlog_file{instance=~\"$instance\", node=\"relay\"} - ON(instance) dm_syncer_binlog_file{instance=~\"$instance\", task=~\"$task\", node=\"syncer\"}",
           "format": "time_series",
           "hide": false,
           "interval": "$interval",
           "intervalFactor": 2,
-          "legendFormat": "{{task}} - {{instance}}",
```

Comment:
I think we should remain `{{task}} - {{instance}}` as each task will produce a different value

Reply:
this metric: `dm_relay_binlog_file`  don't have label `task` so this legend looks like this ` - 127.0.0.1:8621`

```


dm_relay_binlog_file{instance="127.0.0.1:8262", job="dm-worker-1", node="master"} | 1
-- | --
dm_relay_binlog_file{instance="127.0.0.1:8262", job="dm-worker-1", node="relay"}


```

Reply:
ping @glorv  ptal

Reply:
maybe you can use `group_right`

```
dm_relay_binlog_file{instance="172.16.5.177:8101", node="relay"} - ON(instance) group_right dm_syncer_binlog_file{instance="172.16.5.177:8101", node="syncer"}
```

please find a good solution from https://prometheus.io/docs/prometheus/latest/querying/operators/#many-to-one-and-one-to-many-vector-matches

Reply:
due to only syncer_binlog_fie have many to on relationship with relay_binlog_file, only one dashboard need to be adjusted. added in https://github.com/pingcap/tiflow/pull/4405/commits/28f599425c64fd2c4fbd88995c324dd4dfecd228 , preview link is [here](http://172.16.5.177:3000/d/Ja68YqRnzxxxxxxx/dm-monitor-professional-fixed?editPanel=44&orgId=1)